pkg/config/config.go

```diff
@@ -33,6 +33,8 @@ type Config struct {
 	// Internal-only fields
 	Ingress      IngressConfig `json:"-"`
 	userSettings *Config       `json:"-"` // the values read from the config file
+
+	MultiNode MultiNode `json:"multiNode"` // the value read from commond line
```

Comment:
```suggestion
	MultiNode MultiNode `json:"-"` // the value read from commond line
```

I think we will want to hide this structure from the json marshaller, so that the show-config command does not expose it.